.bowerrc

```diff
@@ -0,0 +1,6 @@
+{
+  "ignoredDependencies": [
+    "bootstrap",
+    "bootstrap-sass-official"
```

Comment:
No, it is not in devDepencies so it is not needed at the moment.
but since I ignored bootstrap for the functional ambiguity, I thought it would be needed too.
Do you have any idea to leave bootstrap installed and ignore it only during `wiredep`?
I am not sure ignoring install is the best solution to prevent doubling bootstrap.js during `wiredep`.